include/EGL/eglext_angle.h

```diff
@@ -355,6 +355,11 @@ EGLAPI EGLBoolean EGLAPIENTRY eglSwapBuffersWithFrameTokenANGLE(EGLDisplay dpy,
 #define EGL_EXTERNAL_CONTEXT_SAVE_STATE_ANGLE 0x3490
 #endif /* EGL_ANGLE_external_context_and_surface */
 
+// Shapr3D uses it!
+#define EGL_TEXTURE_SWIZZLING_TYPE 0x35FF
+#define EGL_TEXTURE_SWIZZLING_BGRA 0x35FE
+#define EGL_TEXTURE_SWIZZLING_RGBA 0x35FD
```

Comment:
Even though we don't specify one formally, I think it's advisable to follow the conventions of an ANGLE EGL extension here. So, a macro for `EGL_ANGLE_foo_bar` and all new symbols suffixed with `_ANGLE`.

Reply:
I deliberately want to avoid adding a new extension, because it just gives us a bigger surface to conflict with others. This is a workaround very specific to Shapr, so I'd keep it as tight and small as possible